src/index.html

```diff
@@ -11,6 +11,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search_top">
+      <img class="search_img" src="images/Search.svg"
+       alt="search">
+      <input
+        data-qa="big"
+        class="search_big"
+        type="search"
+        placeholder="Try “Los Angeles”"
+      >
+    </div>
+    <div class="search_down">
+      <img class="search_img_small" src="images/Search.svg"
+       alt="search">
+      <input data-qa="small" class="search_small"
+       type="search" placeholder="Try “Los Angeles”">
```

Comment:
```suggestion
      <input 
        data-qa="small" 
        class="search_small"
        type="search" 
        placeholder="Try “Los Angeles”"
      >
```
Consider fixing indents according to the code style guide